Gemfile.lock

```diff
@@ -2,6 +2,7 @@ PATH
   remote: .
   specs:
     killbill-cybersource (4.0.11)
+      accent_to_ascii (~> 0.1.1)
```

Comment:
> Excuse for my ignorance, but I did not get your point of 'harder to update', would you please explain more?

In case there is a mapping missing, as it is the case for ł (Michał is a common Polish first name), we would need to:

* Create a PR against https://github.com/unieagle/accent_to_ascii
* Have you merge it
* Have you push a new gem
* Update the dependency in this `Gemfile.lock`
* Release this plugin

This is quite a overhead. Shared libraries are useful to avoid duplicated code, but in this specific example, I am having a hard time seeing the value (the gem just has a map?).